Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -252,9 +263,18 @@ extension TabScrollingController: UIScrollViewDelegate {
 
         suppressToolbarHiding = false
     }
-
+    
+    func scrollViewDidEndDecelerating(scrollView: UIScrollView) {
+        showOrHideWebViewContainerToolbar()
+    }
```

Comment:
Limitation:
If we scroll up and let it decelerate until the WebViewControllerToolbar appears, during the "deceleration time", a blank space could appear in the place of the toolbar.